python/taichi/lang/__init__.py

```diff
@@ -566,6 +568,39 @@ def init(arch=None,
                                                 daemon=True)
         check_version_thread.start()
 
+    # Check if installed version meets the requirements.
+    if require_version:
+        # Extract version number part (i.e. toss any revision / hash parts).
```

Comment:
let's still keep this in a separate function?

Reply:
okay